crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -595,16 +605,41 @@ impl PipelineCache {
     pub fn queue_render_pipeline(
         &self,
         descriptor: RenderPipelineDescriptor,
+    ) -> CachedRenderPipelineId {
+        // SAFETY: We are setting `unchecked` to `false`, that's always safe
+        unsafe { self.queue_render_pipeline_ex(descriptor, false) }
+    }
+
+    /// Like [`Self::queue_render_pipeline()`], but disables runtime checks for the shader.
+    ///
+    /// # Safety
+    ///
```

Comment:
This should fully detail the safety invariant by itself instead of linking to wgpu's docs.

Reply:
Where can I find those safety invariants? wgpu doesn't seem to contain any thorough list and just says `[behavior] like indexing out of bounds`.